Libplanet.RocksDBStore/Libplanet.RocksDBStore.csproj

```diff
@@ -22,14 +20,13 @@ Sdk="Microsoft.NET.Sdk">
   <PackageReference Include="Menees.Analyzers.2017" Version="2.0.3">
     <PrivateAssets>all</PrivateAssets>
   </PackageReference>
-  <PackageReference
-      Include="Microsoft.DotNet.Analyzers.Compatibility"
-      Version="0.2.12-alpha">
+  <PackageReference Include="Microsoft.DotNet.Analyzers.Compatibility" Version="0.2.12-alpha">
```

Comment:
This change seems unnecessary.

Reply:
You're right, but every time I change `csproj`, Rider fixes this. So I fixed it. Would it be better to leave it as it is?

Reply:
Ah, makes sense.  👌